cli/command/container/run.go

```diff
@@ -259,10 +259,11 @@ func runRun(dockerCli *command.DockerCli, flags *pflag.FlagSet, opts *runOptions
 // reportError is a utility method that prints a user-friendly message
 // containing the error that occurred during parsing and a suggestion to get help
 func reportError(stderr io.Writer, name string, str string, withHelp bool) {
+	str = strings.TrimSuffix(str, ".") + "."
 	if withHelp {
-		str += ".\nSee '" + os.Args[0] + " " + name + " --help'"
+		str += "\nSee '" + os.Args[0] + " " + name + " --help'"
 	}
-	fmt.Fprintf(stderr, "%s: %s.\n", os.Args[0], str)
+	fmt.Fprintf(stderr, "%s: %s\n", os.Args[0], str)
```

Comment:
Maybe I did not make myself clear. Here is my thought when `withHelp` is true: 
1. `str = strings.TrimSuffix(str, ".") + "."` will make sure that there is always only one period at the end of str;
2. `str += "\nSee '" + os.Args[0] + " " + name + " --help'"` will append `help` details following the str with a warp line;
3. `fmt.Fprintf(stderr, "%s: %s\n", os.Args[0], str)` will add `os.Args[0]` before the str.

Thus I think it will work as my step. Please correct me if I missed something. @AkihiroSuda